projects/dex-ui/src/components/Liquidity/RemoveLiquidity.tsx

```diff
@@ -71,21 +98,35 @@ export const RemoveLiquidity = ({ well, txnCompleteCallback }: RemoveLiquidityPr
         if (!oneTokenQuote) {
           return;
         }
-        removeLiquidityTxn = await well.removeLiquidityOneToken(lpTokenAmount, well.tokens![singleTokenIndex], oneTokenQuote, address);
+        const quoteAmountLessSlippage = oneTokenQuote.quote.subSlippage(slippage);
+        removeLiquidityTxn = await well.removeLiquidityOneToken(
+          lpTokenAmount,
+          well.tokens![singleTokenIndex],
+          quoteAmountLessSlippage,
+          address,
+          undefined,
+          { gasLimit: 1000000 }
```

Comment:
Not sure if we should be setting gasLimit like this here (and a few below too)

Reply:
Well the thing is, I was getting failures without adding a higher gas limit.